src/main/public/js/date-picker.js

```diff
@@ -91,7 +89,7 @@ const datePicker = {
     $.post("/case/" + uuid + "/directions-questionnaire/hearing-dates/date-picker/replace", {
       _csrf: csrf,
       hasUnavailableDates: $("input[name=hasUnavailableDates]:checked").val(),
-      unavailableDates: dates,
+      unavailableDates: dates
```

Comment:
IE11 and Edge apparently complain about trailing commas and comments, where other browsers are more lenient; this is the only example I found in our date-picker integration code.